### ISSUE TITLE
Push a new tag and release

### ISSUE THREAD

**Issue by mfridman** [russellhaering/gosaml2#86]
Thank you for your continued support of this package over the years! 

This package will show up with a vulnerability because the latest tag `v0.6.0` has a dependency on an unpatched version of goxmldsig `<v1.1.0`.

For anyone curious, there was a patch upstream https://github.com/russellhaering/goxmldsig/issues/48 against https://github.com/advisories/GHSA-q547-gmf8-8jr7

To solve this we pinned the latest commit explicitly in our `go.mod`

```
github.com/russellhaering/gosaml2 v0.6.1-0.20210916051624-757d23f1bc28
```

But it'll be nice to avoid special casing this and get a version bump, say `v.0.6.1` or `v0.7.0`. What do you think @russellhaering ?

**Comment by chdigvijay20** [comment]
Can you do this please, @russellhaering?

**Comment by mfridman** [comment]
This should now be resolved with the [v0.7.0 release](https://github.com/russellhaering/gosaml2/releases/tag/v0.7.0). Closing.
